.github/workflows/build_and_test.yaml

```diff
@@ -43,6 +43,10 @@ jobs:
       - run: make build-multiarch
         if: github.event_name == 'pull_request'
 
+      # conformance
```

Comment:
this is def needed but we should add this once we have a better understanding on what works/doesnt
@skriss hoping you can guide on what are all the issues that need to be solved to pass some acceptable conformance (at par with the Quickstart guide)

Reply:
Yes, I opened this PR for not merging this immediately(Some conformance is not acceptable yet).